apps/condo/domains/onboarding/hooks/TourContext/useCompletedTourModals.tsx

```diff
@@ -242,6 +244,18 @@ export const useCompletedTourModals = ({ activeStep, setActiveTourStep, refetchS
                 [TourStepTypeType.Meter]: handleViewGuideClick,
             },
         },
+        createMeter: {
+            availableTourFlow: [TourStepTypeType.Meter],
+            subtitleLinkHref: '/meter',
+            subtitleLinkIcon: Meters,
+            newFeatures: {
+                employee: [TakeReadingsFromResidentsFeatureMessage, ArchvieMetersResidentFeatureMessage],
```

Comment:
any resident features?
for example, after meter was created, resident can submit meter readings in mobile app

Reply:
added  resident new feature, thanks